sdk/anomalydetector/ai-anomaly-detector/review/ai-anomaly-detector.api.md

```diff
@@ -27,19 +27,30 @@ export interface AnomalyContributor {
 }
 
 // @public
-export class AnomalyDetectorClient {
-    constructor(endpointUrl: string, credential: TokenCredential | KeyCredential, options?: AnomalyDetectorClientOptions);
-    deleteMultivariateModel(modelId: string, options?: DeleteMultivariateModelOptions): Promise<void>;
-    detectAnomaly(modelId: string, detectionRequest: DetectionRequest, options?: DetectAnomalyOptions): Promise<AnomalyDetectorClientDetectAnomalyResponse>;
-    detectChangePoint(body: DetectChangePointRequest, options?: DetectChangePointOptions): Promise<AnomalyDetectorClientDetectChangePointResponse>;
-    detectEntireSeries(body: DetectRequest, options?: DetectEntireSeriesOptions): Promise<AnomalyDetectorClientDetectEntireResponse>;
-    detectLastPoint(body: DetectRequest, options?: DetectLastPointOptions): Promise<AnomalyDetectorClientDetectLastPointResponse>;
-    exportModel(modelId: string, options?: ExportModelOptions): Promise<AnomalyDetectorClientExportModelResponse>;
-    getDetectionResult(resultId: string, options?: GetDetectionResultOptions): Promise<AnomalyDetectorClientGetDetectionResultResponse>;
-    getMultivariateModel(modelId: string, options?: GetMultivariateModelOptions): Promise<AnomalyDetectorClientGetMultivariateModelResponse>;
-    listMultivariateModel(options?: ListMultivariateModelOptionalParams): Promise<AnomalyDetectorClientListMultivariateModelResponse>;
-    listMultivariateModelNext(nextLink: string, options?: ListMultivariateModelNextOptionalParams): Promise<AnomalyDetectorClientListMultivariateModelNextResponse>;
-    trainMultivariateModel(modelRequest: AnomalyDetectorClientModelInfo, options?: TrainMultivariateModelOptions): Promise<AnomalyDetectorClientTrainMultivariateModelResponse>;
+export class AnomalyDetectorClient extends AnomalyDetectorClientBase {
+    constructor(endpointUrl: string, credential: TokenCredential | KeyCredential, options?: PipelineOptions);
+}
+
+// @public (undocumented)
+export class AnomalyDetectorClientBase extends AnomalyDetectorClientContext {
+    constructor(endpoint: string, options?: AnomalyDetectorClientOptionalParams);
+    deleteMultivariateModel(modelId: string, options?: coreHttp.OperationOptions): Promise<coreHttp.RestResponse>;
```

Comment:
The deleted method had this type instead `Promise<void>`, should the swagger be updated to keep that return type?

Reply:
I don't think the Generator can do that yet :( I'll keep the generated type and we can discuss if we want to write a manual override for this or implement error swallowing in the generator